GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiableAccountDetail.m

```diff
@@ -17,4 +17,25 @@
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
 
 @implementation GIDVerifiableAccountDetail
+
+- (instancetype)initWithAccountDetailType:(GIDAccountDetailType)accountDetailType {
+   self = [super init];
+   if (self) {
+       _accountDetailType = accountDetailType;
+   }
+   return self;
+}
+
+// Returns dictionary mapping account detail type to scope.
```

Comment:
I don't think this comment is necessary.

Reply:
Done.